lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -613,7 +613,12 @@ class _CanvasPool extends _SaveStackTracking {
         context.save();
         context.filter = 'none';
         context.strokeStyle = '';
-        context.fillStyle = colorToCssString(color);
+        final int red = color.red;
+        final int green = color.green;
+        final int blue = color.blue;
+        // Multiply by 0.4 to make shadows less aggressive (https://github.com/flutter/flutter/issues/52734)
+        final int alpha = (0.4 * color.alpha).round();
+        context.fillStyle = colorComponentsToCssString(red, green, blue, alpha);
         context.shadowBlur = shadow.blurWidth;
         context.shadowColor = colorToCssString(color.withAlpha(0xff));
```

Comment:
Why is this overriding the alpha channel of the shadow color to 0xff?

Can we do this instead?

```
context.shadowColor = colorToCssString(color);
```

(Assuming that color has the right alpha value set?)

Reply:
@hterkelsen, do you recall why this was necessary? FWIW, the shadows look OK to me: https://github.com/flutter/goldens/blob/ae6003206eb721137c20cd56d8d1d8e2a76d6dd1/engine/web/shadows.png